pkg/plugins/runtime/k8s/webhooks/injector/injector.go

```diff
@@ -212,6 +223,10 @@ func (i *KumaInjector) NewSidecarContainer(pod *kube_core.Pod, ns *kube_core.Nam
 				Name:  "KUMA_DATAPLANE_RUNTIME_TOKEN_PATH",
 				Value: "/var/run/secrets/kubernetes.io/serviceaccount/token",
 			},
+			{
+				Name:  "KUMA_CONTROL_PLANE_CA_CERT",
+				Value: i.caCert,
```

Comment:
Is it secure that Kuma CP injects CA cert which will be used to verify Kuma CP? Can someone put their own CA to `CaCertFile` and impersonate `bootstrap` at the end?
Can we use Kubernetes Secrets and mount them into the sidecar?

Reply:
Secrets are Namespaced Scope, we would have to populate this secret all over the namespaces.
CA cert is something public so I see no problem with placing this as an env var.

If someone has a control over Control Plane, it's done. They can do whatever they can.

Reply:
You don't need control over Control Plane, Control Plane reads CA from `CaCertFile` I think it's possible to put malicious CA here. 
Yes, CA cert is something public, but I think you should trust the source who provides you this CA. But in this case, we validate Kuma CP with CA provided by Kuma CP.

Reply:
You need to change CaCertFile AND Bootstrap URL that we are providing for the attack, therefore you need control over the control plane. Or am I missing some cases here?

Reply:
Okay, probably you are right. It just feels weird that the one who you are trying to attest provides you a way how to attest to him. But then I realized it is not Kuma CP provides CA, webhook provides this CA. Injector technically is part of Kubernetes and is trusted by Kubernetes based on certs